modules/configuring-cluster-loader.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/using-cluster-loader.adoc
+
+[id='configuring_cluster_loader_{context}']
+= Configuring Cluster Loader
+
+Create multiple namespaces (projects), which contain multiple templates or pods.
+
+Locate the configuration files for Cluster Loader in the `config/` subdirectory.
+The pod files and template files referenced in these configuration examples are
+found in the `content/` subdirectory.
+
+== Configuration fields
+
+.Top-level Cluster Loader Fields
+|===
+|Field |Description
+
+|`cleanup`
+|Set to `true` or `false`. One definition per configuration. If set to `true`,
+`cleanup` will delete all namespaces (projects) created by Cluster Loader at the
+end of the test.
+
+|`projects`
+|A sub-object with one or many definition(s). Under `projects`, each
+namespace to create is defined and `projects` has several mandatory subheadings.
+
+|`tuningset`
+|A sub-object with one definition per configuration. `tuningset` allows the user
+to define a tuning set to add configurable timing to project or object creation
+(pods, templates, and so on).
+
+|`sync`
+|An optional sub-object with one definition per configuration. Adds synchronization
+possibilities during object creation.
+|===
+
```

Comment:
Ah. I see what you are doing here. The fields for the parameters up there are below. 
Would this topic work as a procedure? 
1. Create projects
fields
2. Define tuningset:
fields
etc
??

Reply:
Maybe the example before the field definitions would make sense.